addons/interaction/XEH_postInit.sqf

```diff
@@ -28,6 +28,11 @@ ACE_Modifier = 0;
     {_lamp setHit [_x select 0, (_x select 1) max _disabledLampDMG];nil} count _hitPointsDamage;
 }] call CBA_fnc_addEventHandler;
 
+[QGVAR(switchCollisionLight), {
+    params ["_target", "_player", "_state"];
+    _player action [["CollisionLightOff", "CollisionLightOn"] select _state, _target];
```

Comment:
Would https://community.bistudio.com/wiki/setCollisionLight be simpler?

Reply:
now I see why I chose action instead of this command. Because of this note:
```
Note that the vehicle has to be local, for global variant use Arma 3 Actions "CollisionLightOn"/"CollisionLightOff"
```
I didn't want to use CBA events and just used these actions.
Could you remove this note from wiki please? I'll replace action with command.

Reply:
Why remove the note? is it invalid?

Reply:
yes, vehicle must be local for both command and action, see https://github.com/acemod/ACE3/pull/6078#pullrequestreview-99120289

Reply:
It might only be a problem for this object simulation type.

Reply:
just checked Buoy, Road Cone, civil plane - all of them require local vehicle